docs/how-to-guides/develop-a-flex-flow/connection-support.md

```diff
@@ -0,0 +1,66 @@
+# Connection support
+
+## Model config in `__init__`
+
+Just like example in [batch run](#batch-run-with-yaml), it's supported to reference connection in ModelConfig.
+And connection will be resolved and flatten connection's fields to ModelConfig.
+**Note**: Currently model config only support OpenAI and AzureOpenAI connection.
+For custom connection, please use [connection in init](#connection-in-__init__).
+
+## Connection in `__init__`
+
+It's also supported to directly pass connection by **name** in `__init__`.
+
+```python
+class MyFlow:
+    def __init__(self, my_connection: AzureOpenAIConnection):
+        pass
+```
+
+Note:
+
+- Union of connection types(`Union[OpenAIConnection, AzureOpenAIConnection]`) is not supported.
```

Comment:
do we have some page introduce the model config?

Reply:
Added a page for it